<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pExit_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pConnectMsg_txt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>441</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="pClearCon_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pClearDisplay_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>510</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="pStart_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>server is not start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pStart_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pStop_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pNibp_rb">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>NIBP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pEcg_rb">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ECG</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pSpo2_rb">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SPO2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="pGloupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="pBox_Label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="pMsg_Txt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>311</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>frequence</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>timeout</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>readcount</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pFre_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pTm_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="pRc_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>410</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pFreOk_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pRcOk_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pFreCancel_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pRcCancel_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>410</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pST_check">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>start to test</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pShowReadData_check">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>show data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pClear_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pExit_btn_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pSendIdCommand">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>send comfire id commond</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pSendData">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>send data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pStopSend">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>stop send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pGenerateData_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>generateData</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="pStatistics_txt">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>270</y>
       <width>301</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="pSendData_check">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>send data</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
